frontend/src/components/regions/UserActions/ActivityDashboardBarGraph.jsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import {
+  BarChart,
+  Bar,
+  XAxis,
+  // YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+} from "recharts";
+
+import { eventTypeColors } from "../../../colors";
+
+const data = [
+  {
+    id: "date=2022-08-12&event_type=1&filter_by_actor_user=2&filter_by_effected_user=",
```

Comment:
This data is in a funny place.  It should be passed to your component from storybook

Reply:
This data also doesn't seem to be the correct data at all. It looks like burndown snapshot data. What we are building here is not another burndown chart.  

If you were to think about your implementation from a UX perspective:
- The activity page already has a burndown chart. It has a separate line per metric. We can see how far each person is at any point in time and we can get an idea of their pace by looking at the gradients
- you now aim to put the same data into a bar chart, and add some colors to it that have nothing at all to do with the colors of the burndown chart above
- A bar chart is just not as powerful as a line graph when representing burndown data. It would add no value to any user on our system
- On top of that, the color coding would not match up with the proper burn-down chart. So the best you can hope to achieve is to confuse people

Please always think things through.

Please look at the api calls that were implemented in the story. They were implemented as a part of this story because they are actually useful in this story. Those things weren't implemented for fun. Tasks in a story are related to other tasks in the story.  

https://www.notion.so/The-Umuzi-Build-Engine-Tilde-Syllabus-9b8df1933b514e369c909d16730ad145?p=811ff06e03b94f35b568fe81bcbac5ca&pm=s

Reply:
Even if you were not aware of the apis then this should have been a red flag:

```
<Bar
        dataKey="reviewsCompleted"
        fill={eventTypeColors.CARD_STARTED}
        date="Reviews Completed"
      />
```

If you ever find yourself randomly guessing in your code then it means you need to stop what yo are doing and spend more time understanding what you are meant to do.   Why on earth should a completed review be the same colour as a CARD_STARTED event?  

We are building a machine. The data needs to line up exactly. If it does not line up exactly then it is a problem that needs to be fixed.